docs/static/releasenotes.asciidoc

```diff
@@ -14,6 +15,27 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-4]]
+=== Logstash 6.1.4 Release Notes
+
+* There are no changes to Logstash core in this release
+
+==== Plugins
+
+*Dissect Filter*
+
+* Fixed bug where multibyte UTF8 strings could cause "Dissector mapping, field found in event but it was empty" errors
+* Fixed bug where, if a dissection is defined with a newline as part of a delimiter, it is ignored.
```

Comment:
* Fixed bug where a dissection is ignored if it is defined with a newline as part of a delimiter.